generators/client/templates/angular/webpack/_webpack.dev.js

```diff
@@ -123,6 +121,11 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
         new webpack.WatchIgnorePlugin([
             utils.root('src/test'),
         ]),
+        new AngularCompilerPlugin({
```

Comment:
why are we doing aot in dev? I dont think AOT is suitable for incremental builds in yarn start

Reply:
also its slower than current setup which is bad in dev as it is already quite slow, we dont want to make it slower

Reply:
I just tried on a sample project from our Travis build and its initial build and yarn start is very slow(~65 sec even in my laptop, so it will be even slower on normal machines) compared to around 30 secs on non-AOT build. 

Also when I make changes to a project the incremental build is taking similar time, as I think AOT recompiles everything, which is very bad for dev, so lets not do this as I dont see any benefit of doing AOT in dev

AOT dev
![image](https://user-images.githubusercontent.com/1107223/33672715-2eaf6782-daab-11e7-8c02-6abc670eccfa.png)


Normal dev
![image](https://user-images.githubusercontent.com/1107223/33672764-52902664-daab-11e7-88f0-a1224d3a6f90.png)

Reply:
Are they same ? 🤔 
<img width="234" alt="screen shot 2017-12-06 at 19 41 37" src="https://user-images.githubusercontent.com/12471122/33678903-85c5493a-dabd-11e7-9604-8a913d2056dc.png">
<img width="224" alt="screen shot 2017-12-06 at 19 41 41" src="https://user-images.githubusercontent.com/12471122/33678904-85f22004-dabd-11e7-8e05-5dcc0e802d02.png">

For me it takes ~22s for dev all the time 🕐  and with loaded entities around 30-40s dev build and it will not give you an AOT output, if I am correct. It will produce a dev build only. 

(This is somewhat similar to what angular/cli does, for both prod and dev they use `@ngtools/webpack` ) 

This needs further investigation, I will revert this

Reply:
> even in my laptop,

😏  (don't they have a sarcasm smiley here)

Reply:
He he, I'm just being technical, as @mraible would vouch for, being an octacore Xeon, Precision runs nodeJs stuff much faster than macs :smile: we could benchmark and see if you dont believe me :wink: 
Anyway I test with the default travis build and 15 entities.

Reply:
Btw in your PR when i tried I saw ngfactory stuff being built so not sure if it doesnt do that

Reply:
see
![image](https://user-images.githubusercontent.com/1107223/33681016-a367f2fc-dac3-11e7-9728-26a1ed81ae99.png)